src/site/content/en/blog/api-for-fast-beautiful-web-fonts/index.md

```diff
@@ -0,0 +1,162 @@
+
+```html
+<link href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap" rel="stylesheet" />
+```
+
+When you request a font from the API, you specify which family or families you would like and, optionally, their weights, styles, subsets, and <a href="https://developers.google.com/fonts/docs/getting_started">many other options</a>. The API will then handle your request in one of two ways:
+
+1. If your request uses common parameters that the API already has files for, it will immediately return CSS to your user, directing them to these files.
+2. If you requested a font with parameters that the API doesn’t currently have cached, it will subset your fonts on the fly, using <a href="https://harfbuzz.github.io/harfbuzz-hb-subset.html">HarfBuzz</a> to do it quickly, and return CSS pointing to them.
+
+## Font files can be large, but they don’t have to be
+
+Web fonts can be large; it’s true. Just a single weight of <a href="https://fonts.google.com/noto/specimen/Noto+Sans+JP">Noto Sans Japanese</a> in WOFF2 is almost 3.4MB—and downloading this to each and every one of your users would be a drag on your page load time. When every millisecond counts and every byte is precious, you want to be sure you’re only loading the data your users need.
+
+The Google Fonts CSS API can create very small font files (called subsets), generated in real time, to serve your users only the text and styles required by your website. Instead of serving up an entire font, you can request specific characters using the <a href="https://developers.google.com/fonts/docs/getting_started#optimizing_your_font_requests">`text`</a> parameter.
+
+```html
+<link href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap&text=RobtMn" rel="stylesheet" />
+```
+
+{% Img src="image/Xyvh8LLq27V5yRjH5iS1dbf64pE2/qCy8MLmblSbhXM4aKbDv.png", alt="A chart with a character count of basic Latin, basic Greek, and extended Greek", width="800", height="219" %}
+
+The CSS API also provides additional web font optimizations to your users automatically, without any API parameters. The API will serve your users CSS files with <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range">`unicode-range`</a> already enabled (<a href="https://caniuse.com/font-unicode-range">if supported by their web browser</a>), so they load the fonts for only the specific characters your website needs.
+
+The unicode-range CSS descriptor is one tool that can now be used to combat large font downloads. This CSS property sets a range of unicode characters that the @font-face declaration contains. If one of these characters is rendered on the page, that font is downloaded. This works well for all kinds of languages, so you can take a font that includes Latin, Greek, or Cyrillic characters and make smaller subsets. In the preceding chart, you can see that if you had to load all three of these character sets, it would be over 600 glyphs.
```

Comment:
```suggestion
The unicode-range CSS descriptor is one tool that can now be used to combat large font downloads. This CSS property sets a range of unicode characters that the `@font-face` declaration contains. If one of these characters is rendered on the page, that font is downloaded. This works well for all kinds of languages, so you can take a font that includes Latin, Greek, or Cyrillic characters and make smaller subsets. In the preceding chart, you can see that if you had to load all three of these character sets, it would be over 600 glyphs.
```